microsoft-edge/hosting/toc.md

```diff
@@ -167,178 +167,208 @@
+####### [IWebView2ExecuteScriptCompletedHandler](webview2/reference/win32/0.8.190/IWebView2ExecuteScriptCompletedHandler.md)
+####### [IWebView2FocusChangedEventHandler](webview2/reference/win32/0.8.190/IWebView2FocusChangedEventHandler.md)
+####### [IWebView2MoveFocusRequestedEventHandler](webview2/reference/win32/0.8.190/IWebView2MoveFocusRequestedEventHandler.md)
+####### [IWebView2NavigationCompletedEventHandler](webview2/reference/win32/0.8.190/IWebView2NavigationCompletedEventHandler.md)
+####### [IWebView2NavigationStartingEventHandler](webview2/reference/win32/0.8.190/IWebView2NavigationStartingEventHandler.md)
+####### [IWebView2NewVersionAvailableEventHandler](webview2/reference/win32/0.8.190/IWebView2NewVersionAvailableEventHandler.md)
+####### [IWebView2NewWindowRequestedEventHandler](webview2/reference/win32/0.8.190/IWebView2NewWindowRequestedEventHandler.md)
+####### [IWebView2PermissionRequestedEventHandler](webview2/reference/win32/0.8.190/IWebView2PermissionRequestedEventHandler.md)
+####### [IWebView2ProcessFailedEventHandler](webview2/reference/win32/0.8.190/IWebView2ProcessFailedEventHandler.md)
+####### [IWebView2ScriptDialogOpeningEventHandler](webview2/reference/win32/0.8.190/IWebView2ScriptDialogOpeningEventHandler.md)
+####### [IWebView2WebMessageReceivedEventHandler](webview2/reference/win32/0.8.190/IWebView2WebMessageReceivedEventHandler.md)
+####### [IWebView2WebResourceRequestedEventHandler](webview2/reference/win32/0.8.190/IWebView2WebResourceRequestedEventHandler.md)
+####### [IWebView2ZoomFactorChangedEventHandler](webview2/reference/win32/0.8.190/IWebView2ZoomFactorChangedEventHandler.md)
+### [.NET](webview2/reference/dotnet/0.9.494-reference-webview2.md)
+#### [Microsoft.Web.WebView2.Core Namespace](webview2/reference/dotnet/0.9.494/namespace-Microsoft.Web.WebView2.Core.md)
+#### [CoreWebView2](webview2/reference/dotnet/0.9.494/Microsoft.Web.WebView2.Core.CoreWebView2.md)
+#### [CoreWebView2Controller](webview2/reference/dotnet/0.9.494/Microsoft.Web.WebView2.Core.CoreWebView2Controller.md)
+#### [CoreWebView2Deferral](webview2/reference/dotnet/0.9.494/Microsoft.Web.WebView2.Core.CoreWebView2Deferral.md)
+#### [CoreWebView2DevToolsProtocolEventReceiver](webview2/reference/dotnet/0.9.494/Microsoft.Web.WebView2.Core.CoreWebView2DevToolsProtocolEventReceiver.md)
+#### [CoreWebView2Environment](webview2/reference/dotnet/0.9.494/Microsoft.Web.WebView2.Core.CoreWebView2Environment.md)
+#### [CoreWebView2EnvironmentOptions](webview2/reference/dotnet/0.9.494/Microsoft.Web.WebView2.Core.CoreWebView2EnvironmentOptions.md)
+#### [CoreWebView2PhysicalKeyStatus](webview2/reference/dotnet/0.9.494/Microsoft.Web.WebView2.Core.CoreWebView2PhysicalKeyStatus.md)
+#### [CoreWebView2Settings](webview2/reference/dotnet/0.9.494/Microsoft.Web.WebView2.Core.CoreWebView2Settings.md)
+#### [EdgeNotFoundException](webview2/reference/dotnet/0.9.494/Microsoft.Web.WebView2.Core.EdgeNotFoundException.md)
+#### Event arguments
```

Comment:
I couldn't find these files in the PR. Do they exist?

Reply:
We don't have a dedicated Event arguments landing page if that's what you were looking for. The other Event argument classes (`Microsoft.Web.WebView2.Core.CoreWebView2*EventArgs.md`) are [here](https://github.com/MicrosoftDocs/edge-developer/pull/627/files/d34a64876a03e90b2d528d1c77d1ae7aacbb996b#diff-890ca2cf15844c6d6332c1b3e8e6074e).

Reply:
Thanks for the reply. Looks like the EventArgs topics aren't linked in the ToC. Is that expected?

Reply:
Thanks for the review. Yes this is expected. Because we don't have a dedicated Event arguments landing page, the topic is not linked to anything. AFAIK it's okay with the ToC.